### ISSUE TITLE
Cauchy association test

### ISSUE THREAD

**Issue by zhengli09** [daiqile96/OTTERS#2]
Hi Qile,

Thanks for developing this awesome tool. I was able to run the "training.py" and "testing.py" scripts. They end up providing me with Fusion p-values from different prediction models, but I did not find the OTTERS p-value from ACAT. Did I miss anything there? Thanks!

Best,
Zheng

**Comment by daiqile96** [comment]
Hi Zheng,

I did not include the code to perform ACAT in the current OTTERS package. Instead, I extracted the FUSION p-values from different methods and used the original ACAT package (https://github.com/yaowuliu/ACAT) to combine them. Here is how I use it:
```
ACAT_with_NA = function(p_vec){
  
  p_vec = p_vec[is.na(p_vec) == F]
  
  return(ACAT(p_vec))
  
}
```
I plan to add an R script to the OTTERS package that extracts FUSION p-values and performs ACAT. I'll keep you updated on my progress.
